samples/tutorials/mnist/README.md

```diff
@@ -0,0 +1,110 @@
+    - Train a MINIST model and export it to GCS
+    - Deploy the exported Tensorflow model on AI Platform prediction service
+    - Test the deployment by calling the end point with test data
+
+## Setups Overview:
+### Prerequisites
+Before you follow the instructions below to deploy your own Kubeflow cluster, you should
+
+- have a [GCP project setup](https://www.kubeflow.org/docs/gke/deploy/project-setup/) for your Kubeflow deployment 
+with you having the [owner role](https://cloud.google.com/iam/docs/understanding-roles#primitive_role_definitions) 
+for the project and with the following APIs enabled:
+    - [Compute Engine API](https://pantheon.corp.google.com/apis/library/compute.googleapis.com)
+    - [Kubernetes Engine API](https://pantheon.corp.google.com/apis/library/container.googleapis.com)
+    - [Identity and Access Management(IAM) API](https://pantheon.corp.google.com/apis/library/iam.googleapis.com)
+    - [Deployment Manager API](https://pantheon.corp.google.com/apis/library/deploymentmanager.googleapis.com)
+    - [Cloud Resource Manager API](https://pantheon.corp.google.com/apis/library/cloudresourcemanager.googleapis.com)
+    - [Cloud Filestore API](https://pantheon.corp.google.com/apis/library/file.googleapis.com)
+    - [AI Platform Training & Prediction API](https://pantheon.corp.google.com/apis/library/ml.googleapis.com)
+- have set up [OAuth for Cloud IAP](https://www.kubeflow.org/docs/gke/deploy/oauth-setup/)
+- have installed and setup [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- have installed [gcloud-sdk](https://cloud.google.com/sdk/)
+
+### Setup Environment
+* Deploy a kubeflow cluster through [CLI](https://www.kubeflow.org/docs/gke/deploy/deploy-cli/)
+    - Download and install kfctl
+    - Create user credentials
+    - Setup environment variables
+    - `NOTE` : The [UI](https://www.kubeflow.org/docs/gke/deploy/deploy-ui/) method of deploying a Kubeflow 
+    cluster does not support Kubeflow v0.7 yet
+
+* Create service account
+```bash
+export SA_NAME = [service account name]
+gcloud iam service-accounts create ${SA_NAME}
+gcloud projects add-iam-policy-binding ${PROJECT_ID} \
+    --member serviceAccount:${SA_NAME}@${PROJECT_ID}.iam.gserviceaccount.com \
+    --role 'roles/owner'
+gcloud iam service-accounts keys create ~/key.json \
+    --iam-account ${SA_NAME}@${PROJECT_ID}.iam.gserviceaccount.com
+```
+
+* Install the lastest version of kfp
+```bash
+pip3 install kfp --upgrade --user
+```
+
+* Deploy kubefow
+```bash
+mkdir -p ${KF_DIR}
+cd $kf_dir
+kfctl apply -V -f ${CONFIG_URI}
+```
+### Running Notebook
+Please not that the above configuration is required for notebook service running outside Kubeflow environment. 
+And the examples demonstrated are fully tested on notebook service for the following three situations:
+- Notebook running on your personal computer
+- Notebook on AI Platform, Google Cloud Platform
```

Comment:
It would be useful to post the link as well to the AI Platform notebook.

Reply:
Sure, added.